web/src/components/chart.js

```diff
@@ -36,6 +43,25 @@ import {
 import Async from "react-async"
 import debounce from "lodash/debounce"
 
+const plotOptions = {
+  pdi: {
+    name: "PDI",
+    abbr: "PDI only",
+    primary: true,
+    color: "rgb(255, 109, 4)",
+  },
+  cases: {
+    name: "Number of cases",
+    abbr: "Add cases",
```

Comment:
Just noticed - these are used primarily in the tooltips, so the tooltips now look like this:

![image](https://user-images.githubusercontent.com/3719547/78805027-e7d47980-798e-11ea-9cbe-72c289f6259e.png)

Reply:
Right. Fixing!

Reply:
Where did this fix end up? I'm seeing it in prod but not on develop

Reply:
https://github.com/voxel51/pandemic51/pull/97